source/location_access/locacc_ganons_castle.cpp

```diff
@@ -17,7 +17,7 @@ void AreaTable_Init_GanonsCastle() {
                  // Exits
                  Entrance(GANONS_CASTLE_LOBBY, { [] { return Dungeon::GanonsCastle.IsVanilla(); } }),
                  Entrance(GANONS_CASTLE_MQ_LOBBY, { [] { return Dungeon::GanonsCastle.IsMQ(); } }),
-                 Entrance(GANONS_CASTLE_GROUNDS, { [] { return true; } }),
+                 Entrance(CASTLE_GROUNDS_FROM_GANONS_CASTLE, { [] { return true; } }),
```

Comment:
```suggestion
                 Entrance(HYRULE_CASTLE_GROUNDS, { [] { return IsChild; }}),
                 Entrance(GANONS_CASTLE_LEDGE, { [] { return IsAdult; }}),
```
Could this work instead? If so we don't need the extra `CASTLE_GROUNDS_FROM_GANONS_CASTLE` area.

Reply:
I had that at one point, and it would work just fine, but from a style perspective personally I like having the intermediate area to handle the age-gating, similar to how the `CASTLE_GROUNDS` area works currently. Although I suppose the reason `CASTLE_GROUNDS` exists is for re-use, whereas `CASTLE_GROUNDS_FROM_GANONS_CASTLE` is only used once (at least for now).

Reply:
It is also useful as the entrance pairs listed in `entrance.cpp` can only point to one region that applies to both ages. So the age gating helps with entrance shuffle too. Otherwise `GANONS_CASTLE_LEDGE` would have to have age checks for every access.
I also think it is more readable as child link is never truly able to be at "ganons castle ledge"

Reply:
Are we OK keeping it as is or do you want me to change it? No rush, just clearing up whether or not this is waiting on me.

Reply:
I'm fine with keeping it as is, so it's on the maintainers now.
I suspect you'll need to run clang format though. We have a workflow you can do it from if you can't yourself.

Reply:
Oh I should have it installed, I'll check on that.